app/routes/applicationForm/$applicantId/index.tsx

```diff
@@ -0,0 +1,397 @@
+import { Button, Checkbox, Container, FormControlLabel, Grid, } from '@mui/material';
+import LabeledTextField from '~/core/components/LabeledTextField';
+import { withZod } from '@remix-validated-form/with-zod';
+import { z } from "zod";
+import { ValidatedForm } from "remix-validated-form";
+import { zfd } from "zod-form-data";
+import SelectField from '~/core/components/FormFields/SelectField';
+
+export const validator = withZod(
+  zfd.formData({
+    email: z.string().email({message: "This field is Required"}).min(1), //For the moment this is for testing, once the login with LinkdIn is implemented, I will take the email with which you are logged in.
+    personalEmail: z.string().email({message: "This field is Required"}).min(1),
+    fullName: z.string().min(1,{message: "This field is Required"}),
+    nationality: z.string().min(1,{message: "This field is Required"}),
+    country: z.string().min(1,{message: "This field is Required"}),
+    dayOfBirth: z.string().min(1,{message: "This field is Required"}), 
+    homeAddress: z.string().min(1,{message: "This field is Required"}),
+    phone: z.string().regex(
+      /^[+]?[(]?[0-9]{3}[)]?[-\s.]?[0-9]{3}[-\s.]?[0-9]{4,6}$/im, 
+      'Invalid phone number',
+    ),
+    universityEmail: z.string().email().optional(),
+    emergencyContactName: z.string().optional(),
+    emergencyContactPhone: z.string().optional(),
+    emergencyRelationship: z.string().optional(),
+    gender: z.string().min(1,{message: "This field is Required"}),
+    englishLevel: z.string().min(1,{message: "This field is Required"}),
+    university: z.string().min(1,{message: "This field is Required"}),
+    campus: z.string().optional(),
+    major: z.string().min(1,{message: "This field is Required"}),
+    semester: z.string().min(1,{message: "This field is Required"}),
+    graduationDate: z.string().min(1,{message: "This field is Required"}), 
+    interest: z.string().min(1,{message: "This field is Required"}),
+    experience: z.string().min(1,{message: "This field is Required"}),
+    cvLink: z.string().url({message: "This field is Required"}).min(1),
+    interestedRoles: z.string().optional(),
+    preferredTools: z.string().optional(),
+    startDate: z.string().min(1,{message: "This field is Required"}), 
+    endDate: z.string().min(1,{message: "This field is Required"}), 
+    hoursPerWeek: z.string().min(1,{message: "This field is Required"}), 
+    howDidYouHearAboutUs: z.string().min(1,{message: "This field is Required"}),
+    participatedAtWizeline: z.string().min(1,{message: "This field is Required"}),
+    wizelinePrograms: z.string().optional(),
+    comments: z.string().optional(),
+  })
+);
+  
+  export default function FormPage() {
+    return (
+    <Container>
+      <img src='/HeaderImage.png' alt='Wizeline' style={{width:"100%"}} />
+      <h1>Application Form</h1>
```

Comment:
You can replace the html like this:
`<Typography component="div"> Application For</Typography>`